components/eamxx/src/share/tests/vertical_remapper_tests.cpp

```diff
@@ -99,7 +97,7 @@ void create_remap_file(const std::string& filename, const int nlevs, const std::
 
   scorpio::register_file(filename, scorpio::FileMode::Write);
 
-  scorpio::register_dimension(filename,"nlevs","nlevs",nlevs);
+  scorpio::register_dimension(filename,"nlevs","nlevs",nlevs, true);
```

Comment:
Are you sure nlevs should be marked as a partitioned dimension?

Reply:
Oh probably not